src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier{
+    public int getRandomBall(){
+        Random random = new Random();
+        return random.nextInt(101) ;
```

Comment:
Consider moving the Random instance to be a class-level static final field to avoid creating a new Random object each time the method is called.